src/border_agent/border_agent.cpp

```diff
@@ -163,6 +165,9 @@ BorderAgent::BorderAgent(otbr::Ncp::RcpHost &aHost, Mdns::Publisher &aPublisher)
     , mBaseServiceInstanceName(OTBR_MESHCOP_SERVICE_INSTANCE_NAME)
 {
     mHost.AddThreadStateChangedCallback([this](otChangedFlags aFlags) { HandleThreadStateChanged(aFlags); });
+    mIsEphemeralKeyEnabled = (otThreadGetVersion() >= kThreadVersion14);
+    otbrLogInfo("Thread Version: %d. ", otThreadGetVersion());
```

Comment:
Do we need this? I think otbr-agent already prints Thread's version on start.

Reply:
accept, thanks~